packages/react-components/react-dialog/src/components/DialogContent/DialogContent.types.ts

```diff
@@ -0,0 +1,21 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type DialogContentSlots = {
+  root: Slot<'div', 'main'>;
+};
+
+/**
+ * DialogContent Props
+ */
+export type DialogContentProps = ComponentProps<DialogContentSlots> & {
```

Comment:
`DIalogSurface` to try to be consistent with the `Popover` ? 

your call, I'm still not a fan of either name :D

Reply:
I don't mind, I like the idea of the suffix `Surface` also.  I'll update it! Thx for the opinion 💪🏼